app/views/components/heading.scala.html

```diff
@@ -23,7 +23,7 @@
   }
 
   @if(title.nonEmpty) {
-    <h1 class="title" id="title">@title</h1>
+    <h1 id="title" class="title">@title</h1>
```

Comment:
why ?

Reply:
Other elements in this component have `id` first, so first to align them. Secondly, I've made a mistake because the order was different for other elements so I thought it would be better to have them all in a simiilar style.

Reply:
I agree with @MaciejRewera , Doesn't matter in the grand scheme of things but ID should probably go first